.github/ISSUE_TEMPLATE.md

```diff
@@ -1,11 +1,16 @@
-- **Cli version:**
-- **Standalone install:** yes/no
-- **Node version:**
-- **Npm version:**
-- **Operating system and architecture:**
+- **balena CLI version:** e.g. 11.2.1 (output of the `"balena version"` command)
+- **Operating system version:** e.g. Windows 10, Ubuntu 18.04, macOS 10.14.5
+- **32/64 bit OS and processor:** e.g. 32-bit Windows on 64-bit Intel processor
```

Comment:
Is this a sensible wording / question? I think there are real-world combinations of 32-bit OS on 64-bit processors, right?